pkg/template/processor.go

```diff
@@ -0,0 +1,156 @@
+package template
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"sort"
+	"time"
+
+	projectv1 "github.com/openshift/api/project/v1"
+	templatev1 "github.com/openshift/api/template/v1"
+	"github.com/openshift/library-go/pkg/template/generator"
+	"github.com/openshift/library-go/pkg/template/templateprocessing"
+	errs "github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
should we add a space to separate and organize our imports?

Reply:
not sure what you mean. All stdlib imports are at the top and there's a blank line to separate the "vendor" libs:
```

import (
	"context"
	"fmt"
	"math/rand"
	"sort"
	"time"

	projectv1 "github.com/openshift/api/project/v1"
	templatev1 "github.com/openshift/api/template/v1"
	"github.com/openshift/library-go/pkg/template/generator"
	"github.com/openshift/library-go/pkg/template/templateprocessing"
	errs "github.com/pkg/errors"
	corev1 "k8s.io/api/core/v1"
	apierrors "k8s.io/apimachinery/pkg/api/errors"
	"k8s.io/apimachinery/pkg/runtime"
	"k8s.io/apimachinery/pkg/runtime/serializer"
	"k8s.io/apimachinery/pkg/types"
	"sigs.k8s.io/controller-runtime/pkg/client"
)
```
if I need to import our own packages, I'll do it in-between the stdlib and the vendor pkgs, with a blankline to separate each group.

Reply:
i was thinking to take it one step further and separate based on orgs for the vendor pkgs. so for instance separating openshift from k8s. but it's minor and i'm unsure if thats normal in the go world...i think i was shown a document on this being the best practice - but i can't find the doc :-) either way is fine.